.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -5,38 +5,106 @@ body:
+        > to `@opentelemetry/sdk-node@0.53.x`, NOT `@opentelemetry/sdk-node@0.x.x`. `0.54.x` **WILL NOT** be picked up
+        > by this constraint due to the way semantic versioning works for `0.x` packages. This is intentional, see
+        > https://semver.org/ for details.
+      label: Updating packages
+      options:
+        - label: I have updated all `@opentelemetry/*` packages but the issue persists
+          required: true
+  - type: dropdown
+    id: esm
+    attributes:
+      description: |
+        **How to find this information:**
+          - check your `package.json`
+            - if you have `"type": "commonjs"` or no `"type"` then you're likely using CommonJS
+            - if you have `"type": "module"`, you are likely using ESM - please ensure you have read the [instructions for ESM users](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/esm-support.md) before opening an issue.
+          - other indicators:
+            - using `.js` or `.cjs` (CommonJS)
+            - using `.mjs` (ESM - please ensure you have read the [instructions for ESM users](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/esm-support.md) before opening an issue.)
+      label: Which module type are you using?
+      multiple: false
+      options:
+        - CommonJS
+        - ESM
+        - I don't know
+  - type: checkboxes
+    id: bundlng
+    attributes:
+      description: |
+        Bundling removes `require` and `import` statements, which results in most `@opentelemetry/instrumentation-*` packages not working as they rely on the presense
+        of these statements to function.
+      label: Bundling and `@opentelemetry/instrumentation-*`packages
+      options:
+        - label: I understand that `@opentelemetry/instrumentation-*` packages are not intended to work in bundles apps (with a few exceptions).
+          required: true
+  - type: markdown
+    attributes:
+      value: |
+        ## Bug Reproduciton
+        > [!IMPORTANT]
+        >  **If you use a third-party package that re-distributes OpenTelemetry, open the bug ticket with that third party unless you can provide steps to reproduce this with pure OpenTelemetry.**
+        >  Digging into third-party distributions of OpenTelemetry is not in scope for this project.
   - type: textarea
     attributes:
       label: What happened?
-      description: Please provide as much detail as you reasonably can.
+      description: |
+        > [!IMPORTANT]
+        > **Please provide as much detail as you reasonably can.**
+        > We all know the complexities of developing with JavaScript/TypeScript. **Be specific** about your setup and **DO NOT** assume that the
+        > person handling your bug report knows how to use your specific combination of tooling. Always provide build instructions and config files.
       value: |
         ## Steps to Reproduce
```

Comment:
this part on your image looks a little weird, not sure if intentional

Reply:
Hmm, do you mean because it looks smaller + the bold text being gray instead of black? :thinking: 
I think that's happening because it's part of the description for the box instead of it being a top-level "markdown" element. 

I don't think I can change this, because that would mean that I'd have to move it before the "What happened" header, which makes it feel somewhat out-of-place. :disappointed:

Reply:
<img width="740" alt="Screenshot 2024-11-05 at 10 14 16 AM" src="https://github.com/user-attachments/assets/7cc5c5a2-852f-4dd9-8695-f570d8fcf02d">

This part looks weird, because the `## Steps` suppose to look like a title, but is just looks like it was added by mistake. Having markdown elements such as `##` showing instead of being normally rendered up looks weird to me.

Now I'm realizing this is suppose to be the placeholder. Is that it? So people can add their info on it? In that case, maybe some changing in formatting, so removing the `##` and the `<!--`, this way is just a message of what you should add, without markdown elements that won't be rendered

Reply:
Hmm, [this is actually how it is today](https://github.com/open-telemetry/opentelemetry-js/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=bug_report.yaml). I was considering splitting it up into multiple textareas, but it ends up looking quite a bit weirder. 🤔 

FWIW it looks better when the user actually clicks into the textarea. It'll become a free-text markdown area with preview like so:

**Before clicking**
![image](https://github.com/user-attachments/assets/ee6b0c9b-dafe-4a68-8917-0ae157b92295)
**After clicking**
![image](https://github.com/user-attachments/assets/07c4cecd-dc51-4159-ab36-97d72d1659f5)

Reply:
Ahh I see! Not too bad then! Thank you for looking into it!